scripts/src/java/org/oppia/android/scripts/license/BUILD.bazel

```diff
@@ -13,3 +13,10 @@ kt_jvm_library(
         "//scripts/src/java/org/oppia/android/scripts/common:maven_dependencies_list_generator",
     ],
 )
+
+kt_jvm_library(
+    name = "license_texts_check_lib",
```

Comment:
Do you also need to update the root scripts build file to point to the correct new location?

Reply:
Done.